plugins/obs-outputs/librtmp/rtmp.c

```diff
@@ -3835,7 +3835,7 @@ RTMP_ReadPacket(RTMP *r, RTMPPacket *packet)
         int tmp;
         if (ReadN(r, (char *)&hbuf[1], 2) != 2)
         {
-            RTMP_Log(RTMP_LOGERROR, "%s, failed to read RTMP packet header 3nd byte",
+            RTMP_Log(RTMP_LOGERROR, "%s, failed to read RTMP packet header 3rd byte",
```

Comment:
Well the `2nd byte` is checked a few lines above in <https://github.com/obsproject/obs-studio/blob/392f143d94e0bc37265889cf18f1b58b28f71d71/plugins/obs-outputs/librtmp/rtmp.c#L3825>

Reply:
The line 3825 is reached when `m_nChannel` is 0. On the other hand, the line 3838, which you are changing, is reached when `m_nChannel` is 1. So, these are exclusive. `ReadN` at the line 3836 is reading 2nd and 3rd bytes.

Reply:
???

the full context is:
```C
        if (ReadN(r, (char *)&hbuf[1], 1) != 1)
        {
            RTMP_Log(RTMP_LOGERROR, "%s, failed to read RTMP packet header 2nd byte",
                     __FUNCTION__);
            return FALSE;
        }
```

so the error is thrown when `ReadN` fails to read one(1) additional byte (which would be the 3rd byte, as the last (2nd) byte is read in line 3812 (and an error is output if that failed=

Reply:
https://github.com/obsproject/obs-studio/blob/392f143d94e0bc37265889cf18f1b58b28f71d71/plugins/obs-outputs/librtmp/rtmp.c#L3812
The byte read by above `ReadN` is the 1st byte, isn't it?

https://github.com/obsproject/obs-studio/blob/392f143d94e0bc37265889cf18f1b58b28f71d71/plugins/obs-outputs/librtmp/rtmp.c#L3836
Then, above `ReadN` should follows the 1st byte's `ReadN` and this is reading 2 bytes. So, these bytes are 2nd and 3rd bytes, aren't they?

Reply:
indeed. i was obviously confused. sorry.

in any case: this PR tries to fix spelling mistakes (like `3nd`) rather than improve the wording in general.

do you think changing `3nd`  is a regression?
do you want me to use `2nd and 3rd`? if so, within the `plugins: Fix typos in log-messages ` commit, or in a separate commit?